demystifying_apis/demystifying_apis.md

```diff
@@ -0,0 +1,316 @@
+
+APIs are great for reproducibility, for obtaining data in a way that can be easily recorded and replicated.  They're also great for automating things that are repetitive, like creating a weekly dashboard.
+
+While APIs are a great way to get data, they aren't the only way.  You can still obtain data manually from most sources, by using a web page to navigate to some resource (like a search function or a downloadable document).  This includes federal agencies.  Most have APIs as well as manual download portals where you can find materials or data to download by hand.
+
+Finally, APIs are great for ensuring data freshness, and getting the latest information in real time from an application.
+
+</div>
+***
+
+Which of the following are ways that you might interact with an API?
+
+[[X]] Visiting a URL that you typed into the address bar of a web browser
+[[X]] Automating a script to go to a URL and get the information from there
+[[X]] Using common, generic methods to pack up and send a message using a programming language
+[[X]] Using custom software written specifically for your API that makes it easier to send a message, from within a programming language
+***
+<div class = "answer">
+
+All of the above are ways you can interact with APIs!  Almost always, your API will give you detailed examples that you can copy and paste as you learn how to use the API.  So, rather than learn how to use APIs in general, you're better off searching for how to use the New York Times API or the Census Bureau API or the Open Data Philly API.
+
+</div>
+***
+
+## Additional Resources
+
+A number of libraries have lists of APIs that are useful for researchers, along with links to how to get started with each API.  The lists below have a number of overlaps, but we're including them all because there are also differences from institution to institution!
+
+* [Temple University](https://guides.temple.edu/APIs)
+* [University of California, Berkeley](https://guides.lib.berkeley.edu/information-studies/apis)
+* [University of Virginia](https://guides.lib.virginia.edu/c.php?g=515084&p=10046079)
+* [University of California, San Diego](https://ucsd.libguides.com/data-statistics/apis)
+
+Some APIs (particularly those linked to social media applications) have changing rules relating to their use, due to monetization of data and data privacy.  Read more in [Shifting landscapes of social media data for research](https://www.timeshighereducation.com/campus/shifting-landscapes-social-media-data-research).
+
+The United States Office of the National Coordinator for Health Information Technology (ONC) is driving an effort to develop and standardize APIs related to electronic health information.  Read more about [their project](https://www.healthit.gov/topic/scientific-initiatives/accelerating-apis-scientific-discovery)!
```

Comment:
```suggestion
If you're interested in reading more about APIs in the biomedical and healthcare spaces, [the United States Office of the National Coordinator for Health Information Technology (ONC) is driving an effort to develop and standardize APIs related to electronic health information](https://www.healthit.gov/topic/scientific-initiatives/accelerating-apis-scientific-discovery). 
```
This is just a suggestion to try to make the link text itself a bit more descriptive, but I don't have super strong feelings about it.